model_api/python/openvino/model_api/models/classification.py

```diff
@@ -154,12 +154,39 @@ def postprocess(self, outputs, meta):
         else:
             result = self.get_multiclass_predictions(outputs)
 
+        raw_scores = np.ndarray(0)
+        if self.output_raw_scores:
+            raw_scores = self.get_all_probs(outputs[self.out_layer_names[-1]])
```

Comment:
```suggestion
            raw_scores = self.get_all_probs(outputs[_raw_scores_name])
```
When I was fixing ModelAPI for direct adapter usage, I noticed that the order of output names may change (which is valid, because the opposite is never promised).

Reply:
Updated